WTWSMS/decisions/WtWSMS_religious_decisions.txt

```diff
@@ -429,17 +431,36 @@ decisions = {
 		
 		effect = {
 			give_nickname = nick_with_immortal_soul
-			activate_title = { title = d_priscillianism status = yes }
+			set_global_flag = veggiepriest
+			activate_title = { title = k_priscillianism status = yes }
 			hidden_tooltip = {
-				create_random_priest = {
+				create_character = {
+					name = "Priscillian"
 					dynasty = none
-					random_traits = yes
+					trait = humble
+					trait = vegetarian
+					trait = zealous
+					trait = mystic
+					race = egyptian_arabic
+					religion = priscillianism
+					culture = ROOT
+					female = no
+					attributes = {
+						martial = 8
+						intrigue = 10
+						learning = 10
+					}
+					historical = yes
 				}
 				new_character = {
-					d_priscillianism = {
+					k_priscillianism = {
```

Comment:
Where is there something that doesn't fit with having Priscillianism as the main branch?

The Bishop of Rome is duke-level in the mod.